wakuv2/waku.go

```diff
@@ -969,7 +974,14 @@ func (w *Waku) processQueue() {
 		case <-w.quit:
 			return
 		case e := <-w.msgQueue:
```

Comment:
is this queue for any messages? or only p2p messages? (i.e pulled from a store node/mailserver)

Reply:
AFAIK this queue is for any messages: pulled from mailserver or P2P.

@richard-ramos could confirm if I am not wrong

Reply:
Yes, it's for any message.

Reply:
Ok, so mind that you will not receive a "mailserver" completed signal until there are no more messages in processing, not just store/mailserver messages. In theory it might never end, if the queue is kept busy.

Reply:
Gotcha, I will exclude the mailserver messages from the `map`

Reply:
> Gotcha, I will exclude the mailserver messages from the map

You mean the opposite right? (the code looks correct)

Reply:
Synced offline, Yeah I mean the opposite, also did a bit of renaming